### PR TITLE
Spelling Updates Batch 4

### PR DESCRIPTION
**Pull request by kingthorin**
Cherry picked from: https://github.com/zaproxy/zap-extensions/pull/3083 and rebased current.

For this one, I reverted the "pluggable" change in quickstart as it was only half done (variable names but not classes, etc). I'm not sure the "pending" change is worth keeping, so I can roll it back if wanted/needed just let me know.

### CODE REVIEW

File: addOns/jython/src/main/java/org/zaproxy/zap/extension/jython/JythonOptionsPanel.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ public void validateParam(Object object) throws Exception {
             if (!modulesPath.exists()) {
                 throw new NoSuchFileException(
                         Constant.messages.getString(
-                                "jython.options.error.modulepath.notexist", modulesPath));
+                                "jython.options.error.modulepath.nonexistent", modulesPath));
```
Comment (thc202) [inline addOns/jython/src/main/java/org/zaproxy/zap/extension/jython/JythonOptionsPanel.java:117]: Same as https://github.com/zaproxy/zap-extensions/pull/4306#discussion_r1050796248

**Comment by thc202** [comment]
Agreed.

File: addOns/portscan/src/main/resources/org/zaproxy/zap/extension/portscan/resources/Messages.properties [hunk 2]
```diff
@@ -362,7 +362,7 @@ ports.port.461                        = DataRampSrv
 ports.port.462                        = DataRampSrvSec
 ports.port.463                        = alpes
 ports.port.464                        = kpasswd
-ports.port.465                        = URL Rendesvous Directory for SSM
+ports.port.465                        = URL Rendezvous Directory for SSM
```
Comment (thc202) [inline addOns/portscan/src/main/resources/org/zaproxy/zap/extension/portscan/resources/Messages.properties:365]: This is the only one worth keeping, see comment `Port descriptions taken …`.

**Comment by kingthorin** [comment]
Done

**Comment by kingthorin** [comment]
Crap, messed up my branches. Gimme a few to fix this.

**Comment by kingthorin** [comment]
Done, again.

**Comment by thc202** [comment]
Thank you!

**Review by thc202 — APPROVED** [review]

File: addOns/jython/src/main/java/org/zaproxy/zap/extension/jython/JythonOptionsPanel.java (hunk 1 above)
Comment (jsoref) [reply addOns/jython/src/main/java/org/zaproxy/zap/extension/jython/JythonOptionsPanel.java]: Similarly, `nonexistent` is a word, whereas the phrase would really be `does not exist` with punctuation of sorts.

Again, no attachment to it, but, just why a spelling set will include it.

**Review by psiinon — APPROVED** [review]

File: addOns/jython/src/main/java/org/zaproxy/zap/extension/jython/JythonOptionsPanel.java (hunk 1 above)
Comment (kingthorin) [reply addOns/jython/src/main/java/org/zaproxy/zap/extension/jython/JythonOptionsPanel.java]: Thanks for clarifying josh, I think the thing here was more that the key name was changed but not the associated text, which seemed a bit incongruent.

**Pull request merged**